content/discrete-math/concepts/congruences/congruences.md

```diff
@@ -0,0 +1,81 @@
+---
+Title: 'Congruences'
+Description: 'Congruences refer to the relationship between two integers a and b when they have the same remainder after
+division by positive integer m that is greater than 1.'
+Subjects:
+  - 'Computer Science'
+  - 'Discrete Math'
+Tags:
+  - 'Arithmetic'
+  - 'Math'
+  - 'Logical'
+  - 'Logic'
+CatalogContent:
+  - 'computer-science'
+  - 'paths/computer-science'
+---
+
+**Congruences** refer to the relationship between two integers, _a_ and _b_, that have the same remainder after division
```

Comment:
Doesn't matter much to me if we use `n` (traditional) `m` or any other character to represent the integer so long as it is consistent throughout this. I made everything _m_ but if there's one that I missed feel free to self-revise

Reply:
Hello, I just checked and all the integers are represented by m. Thanks.